modules/rds-dashboard/main.tf

```diff
@@ -2,404 +2,393 @@ terraform {
   required_providers {
```

Comment:
This has been reviewed. And necessary changes have been made to pass github actions CI and local terraform apply.